lib/graphql/base_type.rb

```diff
@@ -158,5 +158,18 @@ def edge_type
     def define_edge(**kwargs, &block)
       GraphQL::Relay::EdgeType.create_type(self, **kwargs, &block)
     end
+
+    # Return a GraphQL string for the type definition
+    # @param schema [GraphQL::Schema]
+    # @param printer [GraphQL::Schema::Printer]
+    # @param context [Hash]
+    # @param only [<#call(member, ctx)>]
+    # @param except [<#call(member, ctx)>]
+    # @param warden [GraphQL::Warden]
+    # @return [String] type definition
```

Comment:
@rmosolgo do you want me to enumerate out the params here like I did? Or just put `args` with a comment about it?